[0.4.0] Compilation error on Windows 64 bits

on 32 bits, Eip register is used while Rip should be used on 64 bits

see https://github.com/Libvisual/libvisual/blob/master/libvisual/libvisual/lv_cpu.c#L136

libvisual 0.4.0 : mingw-w64 with MSYS2

./configure --host=x86_64-w64-mingw32 

btw, i tweak a bit configure because i also have this error : 

checking for the appropriate definition for size_t... configure: error: No type matching size_t in size

this is because on Windows 64 bits, the 'long' type is 32 bits, contrary to UNIX (see https://en.wikipedia.org/wiki/64-bit_computing#64-bit_data_models)

--- a/libvisual/libvisual/lv_cpu.c
+++ b/libvisual/libvisual/lv_cpu.c
@@ -133,7 +133,11 @@ static void sigfpe_handler_sse (int signal, struct sigcontext sc)
 LONG CALLBACK win32_sig_handler_sse (EXCEPTION_POINTERS* ep)
 {
 	if (ep->ExceptionRecord->ExceptionCode == EXCEPTION_ILLEGAL_INSTRUCTION) {
+#ifdef _AMD64_
+		ep->ContextRecord->Rip +=3;
+#else
 		ep->ContextRecord->Eip +=3;
+#endif
 		cpu_caps.hasSSE = FALSE;
 		return EXCEPTION_CONTINUE_EXECUTION;
 	}

